### DISCUSSION TITLE
getbootstrap.com is looking broken for me

### DISCUSSION THREAD

**Discussion by Schmell** [twbs/bootstrap#35118] [General]
maybe there is something wrong with my machine, but https://getbootstrap.com is not rendering correctly and this is shortly after updating chrome

Is anyone else seeing this?

<img width="1440" alt="boostrapScreenshot" src="https://user-images.githubusercontent.com/4114625/136061804-e9cdea55-63b6-4b80-8bd0-2933e538a7f3.png">

here is the first error in the console

getbootstrap.com/:20 Failed to find a valid digest in the 'integrity' attribute for resource 'https://getbootstrap.com/docs/5.1/dist/css/bootstrap.min.css' with computed SHA-256 integrity 'sAcc18zvMnaJZrNT4v8J0T4HqzEUiUTlVFgDIywjQek='. The resource has been blocked.

**Comment by matintrenta** [comment]
I have the same problem, Im on Windows 10 and Microsoft Edge.
I was working with bootstrap when this problem occurs.

**Comment by ffoodd** [comment]
We juste released 5.1.2, maybe clear your browser cache?

**Reply by Schmell** [reply]
It appears to be fine now
